src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx

```diff
@@ -390,4 +390,7 @@
   <data name="DoNotCopyValueNoReturnValueFromReferenceMessage" xml:space="preserve">
     <value>Cannot return a value from a reference to non-copyable type '{0}'</value>
   </data>
+  <data name="ApplyTraitToContainingType" xml:space="preserve">
+    <value>Apply trait to containing type</value>
```

Comment:
Consider `'Move trait etc.'` to make it clearer that it's going to move these and remove from all methods?

Reply:
I used 'Apply' for one main reason: this refactoring _does not_ verify that all test methods use the trait consistently before offering to apply it. In other words, it is possible for this refactoring to add traits to tests which did not previously have the trait.

Reply:
➡️ Not planning to change this for now, but see previous comment for details